scripts/check_frontend_test_coverage.py

```diff
@@ -150,12 +161,43 @@ def check_not_fully_covered_filenames_list_is_sorted() -> None:
         sys.exit(1)
 
 
-def check_coverage_changes() -> None:
+def check_if_file_should_be_checked(
+    file_path: str,
```

Comment:
Confusing since this can also be a filename. Consider only supporting either filenames or file paths. Supporting both makes the code more complicated, and I don't see why both are necessary

Reply:
Changed everything to file paths.